exercises/rational-numbers/README.md

```diff
@@ -0,0 +1,27 @@
+A rational number is defined as the quotient of two integers `a` and `b`, called the numerator and denominator, respectively, where `b != 0`.
```

Comment:
This README looks a little incomplete.

If you use the [configlet](https://github.com/exercism/configlet/) tool, it will generate the README for you. Normally for our track, we have a couple of added bits that configlet will automatically pull in.

The [script](https://github.com/exercism/java/blob/master/bin/fetch-configlet) should pull down configlet for you.

Reply:
That's great; I didn't know about that. I'll check it out.